pkgs/by-name/ni/niri/package.nix

```
@@ -116,7 +119,7 @@ rustPlatform.buildRustPackage rec {
   meta = {
     description = "Scrollable-tiling Wayland compositor";
     homepage = "https://github.com/YaLTeR/niri";
-    changelog = "https://github.com/YaLTeR/niri/releases/tag/v${version}";
+    changelog = "https://github.com/YaLTeR/niri/releases/tag/${finalAttrs.src.tag}";
```

```suggestion
    changelog = "https://github.com/YaLTeR/niri/releases/tag/${toString finalAttrs.src.tag}";
```
`src.tag` may be `null` if `src` is overriden

---

is `toString` any good if it's null? that will cause an empty interpolation, and point us at [a 404 page](https://github.com/YaLTeR/niri/releases/tag/).?

---

The point of toString was to avoid any evaluation errors; there's no guarantee we'll actually be getting anything tag-related from `src` or `version`, so this attribute evaluating to 404'd URLs is inevitable

---

What if `src` doesn't have a `tag` at all? In that case we would still get an evaluation error, right? I think I will just change it back to `v${finalAttrs.version}` then

---

Yeah, that would be true in the case of other fetchers being used. Didn't think of that!